web/src/components/participants/ParticipantList.svelte

```diff
@@ -3,27 +3,32 @@
   import { client } from "../../lib/apollo";
   import { participantPerQueryType } from "../../lib/models/participants/participants.js";
   import { handleErrorMessage } from "../../utils/errorQuery";
+  import Pagination from "../base/Pagination.svelte";
   import DataCell from "./DataCell.svelte";
 
   export let queryArgs;
   export let type = "run";
+  export let limit = 50;
+  export let offset = 0;
```

Comment:
I don't know if we want to bind offset up to the parent component or keep all of this offset changing logic in this component. I think it should be easy enough to modify the queryArgs dynamically in this component without having to move the offset into the parent and then back down into this component through the queryArgs. I think it's a bit cleaner if the parent doesn't set offset or limit in the queryArgs at all, and that's managed here (although limit could still stay an export here, so the parent can decide the size of a page). This way, if we do move to relay style pagination down the road, there's less to untangle. 👍